src/events/masp.upsert.service.ts

```diff
@@ -191,8 +197,11 @@ export class MaspTransactionsUpsertService {
       return maspTransactions;
     });
 
```

Comment:
Had to add additional query here for disconnecting operations, when transactions aren't sent with disconnecting block, we can no long use the users calculated from `operation`.